mind_wave.py

```diff
@@ -159,11 +158,14 @@ def chat_parse_content(self, buffer_content):
         content = ''  # initialize the content
 
         for line in lines:
-            if line.startswith('------ '):
+            if line.startswith('# > ') or line.startswith('## > '):
```

Comment:
Should use 

```
if line == "# > User:" or line == "## > Assistant:":
```

Reply:
I prefer chat content is right after the markers, not on a new line. With this, we can still see the first line of chat when we collapse the nodes:

Example:
- full display
![image](https://user-images.githubusercontent.com/10945297/231454992-8544e412-3067-46c6-88fd-c11216e222e6.png)
- chat nodes collapsed
![image](https://user-images.githubusercontent.com/10945297/231455264-d7d537f2-873c-4f93-8123-a405f5f5e403.png)
- all collapses
![image](https://user-images.githubusercontent.com/10945297/231455534-268e7bc0-5c9e-4c46-96b7-851527e2e4b8.png)
- imenu
![image](https://user-images.githubusercontent.com/10945297/231455791-3a19d33b-ae76-43ef-8d96-ced830c87838.png)

Reply:
Please add convert function before mind-wave-call-async "chat_ask" to help user convert old marker to new marker.

This patch make old *.chat file can't work, i won't merge this patch if convert function not provide, sorry.

Reply:
sure I will do it when I have time, thanks

Reply:
done, added a convert function and and a flag to run it.
then the flag is true which is the default value, the function gets run on chat mode enabled and insde `mind-wave-chat-ask-send-buffer`